### PR TITLE
Restore-DbaDatabase - improve warning output for standby mode databases

### PR DESCRIPTION
**Pull request by 0x7FFFFFFFFFFFFFFF**
Updated the recovery logic to include databases in standby mode. Previously, the code only addressed databases in the restoring state, which did not account for standby scenarios. This change ensures a more comprehensive recovery process.

## Please read -- recent changes to our repo
On November 10, 2022, [we removed some bloat from our repository (for the second and final time)](https://github.com/dataplat/dbatools/issues/8542). This change requires that all contributors reclone or refork their repo.

PRs from repos that have not been recently reforked or recloned will be closed and @potatoqualitee will cherry-pick your commits and open a new PR with your changes.

 - [v] Please confirm you have the smaller repo (85MB .git directory vs 275MB or 110MB or 185MB .git directory)

## Type of Change

 - [x] Bug fix (non-breaking change, fixes # )
 - [ ] New feature (non-breaking change, adds functionality, fixes # )
 - [ ] Breaking change (affects multiple commands or functionality, fixes # )
 - [ ] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1`)
 - [ ] Adding code coverage to existing functionality
 - [ ] Pester test is included
 - [ ] If new file reference added for test, has is been added to github.com/dataplat/appveyor-lab ?
 - [ ] Unit test is included
 - [ ] Documentation
 - [ ] Build system

### CODE REVIEW

File: public/Restore-DbaDatabase.ps1 [hunk 1]
```diff
@@ -650,7 +650,8 @@ function Restore-DbaDatabase {
                     Write-Message -Message "$Database does not exist on $RestoreInstance" -level Warning
                     continue
                 }
-                if ($RestoreInstance.Databases[$Database].Status -ne "Restoring") {
+
+                if (@("Restoring", "Normal, Standby") -notcontains $RestoreInstance.Databases[$Database].Status) {
                     Write-Message -Message "$Database on $RestoreInstance is not in a Restoring State" -Level Warning
```
Comment (niphlod) [inline public/Restore-DbaDatabase.ps1:655]: the message here is not related anymore to the "Restoring" state, so it needs to convey the reason why it's skipped

**Review by niphlod — CHANGES_REQUESTED** [review]

File: public/Restore-DbaDatabase.ps1 (hunk 1 above)
Comment (0x7FFFFFFFFFFFFFFF) [reply public/Restore-DbaDatabase.ps1:655]: Good catch. I'll update it.

Comment (0x7FFFFFFFFFFFFFFF) [reply public/Restore-DbaDatabase.ps1]: Done.

**Review by niphlod — APPROVED** [review]

**Review by wsmelton — CHANGES_REQUESTED** [review]

**Comment by wsmelton** [comment]
This is marked as being a bug fix, what bug issue is being addressed 😉 ?

**Comment by 0x7FFFFFFFFFFFFFFF** [comment]
> This is marked as being a bug fix, what bug issue is being addressed 😉 ?

There doesn't appear to be a related issue; however, it's unable to recover standby database.

**Comment by jpomfret** [comment]
@wsmelton - do you want an issue created to track this bug fix? or we good to merge?

**Comment by potatoqualitee** [comment]
Merging due to Niph's approval since it's been a while.

**Pull request merged**